wannierberri/__evaluate.py

```diff
@@ -124,7 +124,11 @@ def evaluate_K(func,system,grid,fftlib='fftw',
             file_Klist+=".pickle"
     cnt_exclude=0
 
-    print ("using NKdiv={}, NKFFT={}, NKtot={}".format( grid.div,grid.FFT,grid.dense))
+    # TODO : implement and use the repr() or str() method of grid/path
+    try:
+        print ("using NKdiv={}, NKFFT={}, NKtot={}".format( grid.div,grid.FFT,grid.dense))
```

Comment:
Sorry, I found this small bug before, and fixed in PR sym_wann. Then forget it totally

Like give the div of the path in path.py
```
        self.div = np.shape(self.K_list)[0]
```

Reply:
Where do I put this line?